code/modules/spells/spell_types/touch/scream_for_me.dm

```diff
@@ -16,20 +16,49 @@
 	hand_path = /obj/item/melee/touch_attack/scream_for_me
 
 /datum/action/cooldown/spell/touch/scream_for_me/on_antimagic_triggered(obj/item/melee/touch_attack/hand, mob/living/victim, mob/living/carbon/caster)
-	caster.visible_message(
-		span_warning("The feedback mutilates [caster]'s arm!"),
-		span_userdanger("The spell bounces from [victim]'s skin back into your arm!"),
+
+
+	// Informs the wizard of the backfire
+	to_chat(victim, span_warning("The feedback mutilates [caster]'s arm!"))
+	to_chat(caster, span_userdanger("The spell bounces from [victim]'s skin back into your arm!"))
+
+	var/list/wounds = list(
+		WOUND_SEVERITY_TRIVIAL,
```

Comment:
You're adding trivial for what reason?

Reply:
this will just make the spell fail occasionally, trivial slash wounds literally dont exist

Reply:
-and the arg youre putting this into is the maximum, meaning youll only ever accept a non-existant wound 33% of the time